core/templates/pages/contributor-dashboard-page/translation-language-selector/review-translation-language-selector.component.ts

```diff
@@ -92,12 +92,7 @@ export class ReviewTranslationLanguageSelectorComponent implements OnInit {
 
     this.contributionOpportunitiesBackendApiService
       .getPreferredTranslationLanguageAsync()
-      .then((preferredLanguageCode: string|null) => {
-        if (preferredLanguageCode) {
-          this.populateLanguageSelection(
-            preferredLanguageCode);
-        }
-      });
+      .then((_: string|null) => {});
```

Comment:
This doesn't look right. We should still populate the language if the `languageSelection` is not set to the default.

Reply:
Oh, you are right. I change the `populateLanguageSelection()`. Now, we still populate the language but do not call the backend when initializing the selection.

Reply:
@qinghaoyang How do the recent changes fix the issue noted in https://github.com/oppia/oppia/pull/18649#discussion_r1294870058? We should also add tests to verify the changes.

Reply:
I meant to only set the preferred language on line 97 if `languageSelection == 'Language'` (the default). Does that work?

Reply:
Got it.